arch/arc/core/prep_c.c

```diff
@@ -105,7 +105,7 @@ static void adjust_vector_table_base(void)
 	}
 }
 
-extern FUNC_NORETURN void _Cstart(void);
+extern FUNC_NORETURN void z_Cstart(void);
```

Comment:
I second Andy's comment to use z_cstart() and not using camelcase.

Reply:
Yes, fixed.